src/motor-interface-pru.p

```diff
@@ -28,6 +28,7 @@
 #define CONST_PRUDRAM	   C24
 
 #define QUEUE_ELEMENT_SIZE (SIZE(QueueHeader) + SIZE(TravelParameters))
+#define QUEUE_END QUEUE_LEN * QUEUE_ELEMENT_SIZE
```

Comment:
Always put parenthesis around expressions you declare in defines, you never know in which operator precendence context it is used.